README.md

```diff
@@ -1 +1,137 @@
-# FIXME
+<div align="center">
+<img src="https://pl-public-data.s3.amazonaws.com/assets_lightning/Lit_LLaMA_Badge3x.png" alt="Lit-LLaMA" width="128"/>
+
+# ⚡ Lit-StableLM
+
+<!--
+<p align="center">
+  <a href="https://www.lightning.ai/">Lightning.ai</a> •
+  <a href="https://lightning.ai/docs/pytorch/stable/">PyTorch Lightning</a> •
+  <a href="https://lightning.ai/docs/fabric/stable/">Fabric</a>
+</p>
+-->
+
+![cpu-tests](https://github.com/lightning-AI/lit-stablelm/actions/workflows/cpu-tests.yml/badge.svg) <!-- [![Build Status](https://dev.azure.com/Lightning-AI/lit%20Models/_apis/build/status%2FLightning-AI.lit-StableLM?branchName=main)](https://dev.azure.com/Lightning-AI/lit%20Models/_build/latest?definitionId=49&branchName=main) --> [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Lightning-AI/lit-stablelm/blob/master/LICENSE) [![Discord](https://img.shields.io/discord/1077906959069626439?style=plastic)](https://discord.gg/VptPCZkGNa)
+
+<!-- <img src="https://pl-public-data.s3.amazonaws.com/assets_lightning/Llama_pineapple.gif" alt="Lit-LLaMA and pineapple pizza" width="500px"/> -->
+
+</div>
+
+# ⚡ Lit-StableLM
+
+Hackable implementation of the [StableLM](https://github.com/Stability-AI/StableLM) and [Pythia](https://github.com/EleutherAI/pythia) family of models released under the **Apache 2.0 license.**
+
+This implementation builds on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡.
+
+Weights are available under the Apache 2.0 license and can be downloaded [following these instructions](howto/download_weights.md).
+
+## Design principles
+
+This repository follows the **Lit design principle** of **openness through clarity**.
```

Comment:
As a new reader, it's not clear to me that a "lit design principle" is a properly defined term

Maybe this?

```suggestion
This repository follows the **design principle** of **openness through clarity**.
```

Reply:
Fixed it